code/API_definitions/population-density-data.yaml

```
@@ -110,18 +110,17 @@ info:
     # Further info and support
 
     (FAQs will be added in a later version of the documentation)
-  termsOfService: http://swagger.io/terms/
-  contact:
-    email: project-email@sample.com
+    
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.1.0
+  version: 0.1.1-rc1.1

```

Two options here from my point of view:

v0.1.1-rc.1 (first release candidate of v0.1.1)
or
v0.1.1-rc.3 (as there were theoretically already two release candidates of v0.1.0 and only with continuing the number we would get a unique url for this release candidate)

Or directly v0.1.1 as this is only a patch release of v0.1.0 with minor changes and no release candidate is needed ... but then also several other place need to be changed accordingly. 


```suggestion
  version: 0.1.1-rc.1

```